modes/dired/evil-collection-dired.el

```diff
@@ -186,7 +186,17 @@
     ";d" 'epa-dired-do-decrypt
     ";v" 'epa-dired-do-verify
     ";s" 'epa-dired-do-sign
-    ";e" 'epa-dired-do-encrypt))
+    ";e" 'epa-dired-do-encrypt)
+
+    ;; dired-subtree commands
+    (with-eval-after-load 'dired-subtree
+	(evil-collection-define-key 'normal 'dired-mode-map
+	    "I" 'dired-subtree-toggle
+	    (kbd "TAB") 'dired-subtree-cycle
```

Comment:
Does dired-subtree bind TAB on its own?

I just looked into my own bindings and see that I use dired-subtree-toggle instead of cycle for both my dired and dired-sidebar setup.

Reply:
`dired-subtree` doesn't bind any keys on its own, it just provides a set of 16 functions.

Reply:
Hmnn wonder if I should switch to cycle instead of toggle for dired-sidebar. FWIW, they're about the same in functionality. I'm guessing something about wrapping cycle makes dired-sidebar unable to unnest all directories which makes cycle the same as toggle.

This is OK to me.